components/automate-chef-io/content/docs/backup.md

```diff
@@ -132,6 +132,49 @@ The following IAM policy describes the basic permissions Chef Automate requires
 }
 ```
 
+## Backup to GCS (Google Cloud Storage Bucket)
+
+
+Backups can be stored in an existing GCS bucket. You will need to [generate a service account key](https://cloud.google.com/iam/docs/creating-managing-service-account-keys) with the `storage.admin` permission for the project the bucket is associated with.
+
+The following settings are supported:
+
+```toml
+[global.v1.backups]
+  location = "gcs"
+[global.v1.backups.gcs.bucket]
+  # name (required): The name of the bucket
+  name = "<bucket name>"
+
+  # base_path (optional):  The path within the bucket where backups should be stored
+  # If base_path is not set, backups will be stored at the root of the bucket.
+  base_path = "<base path>"
+
+[global.v1.backups.gcs.credentials]
+# This is the JSON credentials file you generate during service account 
+# creation, you must copy/paste the entire contents here (this is just an example)
+json = '''
+  {
+  "type": "service_account",
+  "project_id": "my-favorite-project",
+  "private_key_id": "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+  "private_key": "-----BEGIN PRIVATE KEY-----\nXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX\n-----END PRIVATE KEY-----\n",
+  "client_email": "chef@my-favorite-project.iam.gserviceaccount.com",
+  "client_id": "XXXXXXXXXXXXXXXXXXXXX",
+  "auth_uri": "https://accounts.google.com/o/oauth2/auth",
+  "token_uri": "https://oauth2.googleapis.com/token",
+  "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
+  "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/chef%40my-favorite-project.iam.gserviceaccount.com"
+}
+'''
+```
+
+### GCS Permissions
+
+Per [Elasticsearch's documentation](https://www.elastic.co/guide/en/cloud-enterprise/current/ece-configure-gcp-snapshotting.html#ece_set_up_your_service_account_credentials_2), the service account that is used to backup should be granted the role of 
```

Comment:
@danielcbright The content of this section is repeated from line 138, and does not parallel the format of the S3 section (line 98), which describes the IAM policy and the basic permissions Chef Automate requires to run backup and restore operations.

If it's okay with you, I'm inclined to delete this section and keep the relevant permission info in the Backup to GCS section.

Reply:
I'm ok with it! 💯